guides/Web_Frameworks/Nuxt-js.md

```diff
@@ -0,0 +1,45 @@
+## OVERVIEW:
+
+Nuxt.js is a free and open source web application framework which is based on Vue.js, Node.js, Express.js, Webpack and Babel.js.Its main feature is UI rendering while abstracting away the client/server distribution.This framework is used for creating universal applications.Now the question arises-What are Universal aaplications? An universal app is a mix between client side and server side app. To be an universal app the application must use the same code both on client and server side to render the application components.
+Our goal is to create a framework flexible enough that you can use it as a main project base or in addition to your current project based on Node.js.
+
+Nuxt.js presets all the features that are needed to make your development of a Vue.js Application Server Rendered more enjoyable. It also consists of another deployment feature called nuxt generate which helps in building a static generated Vue.js Application.
+
+#### FEATURES OF NUXT.JS:
+- Write Vue Files (*.vue)
+- Automatic Code Splitting
+- Server-Side Rendering
+- Powerful Routing System with Asynchronous Data
+- Static File Serving
+- ES6/ES7 Transpilation
+- Bundling and minifying of your JS & CSS
+- Hot module replacement in Development
+- Pre-processor: Sass, Less, Stylus, etc.
+- HTTP/2 push headers ready
+- Extending with Modular architecture
+
+#### INSTALLATION:
+Install it with vue-cli:
+
+$ vue init nuxt-community/starter-template {project-name}
```

Comment:
1) If possible can you highlight this command something like ```vue init nuxt-community/starter-template ``` and also, below mentioned commands too!
Use [cheatsheet](https://help.github.com/articles/basic-writing-and-formatting-syntax/) to know more!